rosdep/python.yaml

```diff
@@ -4251,7 +4251,7 @@ python-tinydb-pip:
 python-tk:
   arch: [python2, tk]
   debian: [python-tk]
-  fedora: [blt, tcl, tix, tk]
+  fedora: [python-tkinter]
```

Comment:
Where's a good place to look up fedora packages? I see `python2-tkinter` here:
https://pkgs.org/download/python2-tkinter

Reply:
I usually use https://apps.fedoraproject.org/packages

pkgs.org is a 3rdparty aggregator, it's usually pretty accurate though.

You're right that it's called python2-tkinter

https://apps.fedoraproject.org/packages/python2-tkinter